src/app/static/src/js/components/runWorkflow/runWorkflowProgress.vue

```diff
@@ -1,19 +1,208 @@
 <template>
-    <p>Run workflow progress is coming soon</p>
+    <div
+        class="container mt-3"
+        id="workflow-progress-container"
+        v-if="workflowRunSummaries"
+    >
+        <div class="row mb-3">
+            <label for="workflows" class="form-label col">Workflow</label>
```

Comment:
`for` needs to identify an element by ID, so you can either change the `id` of the <v-select> to `workflows` or (less nice) do `<label for="v-select" ...>`